web/js/function-highlight-addon.js

```diff
@@ -0,0 +1,176 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
If this has a third-party license, I have to ask you to move it to the third_party license, in a subdirectory with the license that applies to it.  Then symlink it from web/js